data/chuwi-minibookx.tablet

```diff
@@ -0,0 +1,17 @@
+# This is for the CHUWI MiniBook X
+#
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/241
+
+[Device]
+Name=Chuwi Minibook X
+Class=ISDV4-aes
+DeviceMatch=i2c:27C6:011A
+Width=23
+Height=14
```

Comment:
is that correct? The width and height are in inches (unfortunately), so this seems ... too big for something called minibook.

Reply:
Oh gosh, my apologies. I thought it was in centimeters. Converted this would be:

```suggestion
Width=9
Height=6
```

(Note: Technically 14cm is about 5.5", but I think you'd probably prefer to round that up)

Reply:
yeah, unfortunately here we use inches. Rounding up is fine, it's not like anyone actually uses this value :) Thanks for the quick update.

Reply:
My pleasure! Thanks for taking a look at this PR, I really appreciate it!